corehq/apps/analytics/tasks.py

```diff
@@ -212,7 +214,10 @@ def track_app_from_template_on_hubspot(webuser, cookies, meta):
 
 @task(queue="background_queue", acks_late=True, ignore_result=True)
 def track_clicked_deploy_on_hubspot(webuser, cookies, meta):
-    _send_form_to_hubspot(HUBSPOT_CLICKED_DEPLOY_FORM_ID, webuser, cookies, meta)
+    ab = {
+        'a_b_variable_deploy': 'A' if deterministic_random(webuser.username + 'a_b_variable_deploy') > 0.5 else 'B',
```

Comment:
no biggie.

On Fri, Mar 25, 2016 at 3:50 PM, Cal Ellowitz <notifications@github.com>
wrote:

> In corehq/apps/analytics/tasks.py
> <https://github.com/dimagi/commcare-hq/pull/10939#discussion_r57445121>:
>
> > @@ -212,7 +214,10 @@ def track_app_from_template_on_hubspot(webuser, cookies, meta):
> >
> >  @task(queue="background_queue", acks_late=True, ignore_result=True)
> >  def track_clicked_deploy_on_hubspot(webuser, cookies, meta):
> > -    _send_form_to_hubspot(HUBSPOT_CLICKED_DEPLOY_FORM_ID, webuser, cookies, meta)
> > +    ab = {
> > +        'a_b_variable_deploy': 'A' if deterministic_random(webuser.username + 'a_b_variable_deploy') > 0.5 else 'B',
>
> Im hoping to pull all of these out of the code, in an overhaul of the way
> we set up AB tests, soon, but happy to pull this one out in the meantime if
> you prefer.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10939/files/31983bc0973393200aeb6fa3ab7e2695fb189007#r57445121>
>